api/client/stats_helpers.go

```diff
@@ -150,7 +154,7 @@ func (s *containerStats) Collect(cli client.APIClient, streamStats bool, waitFir
 				s.mu.Lock()
 				s.err = err
 				s.mu.Unlock()
-				return
+				continue
```

Comment:
@cpuguy83 what do you think about this?  I'm not sure why we're treating a "no response within 2 seconds" differently from an error case. In both cases we don't have data so showing --'s seems better than potentially misleading people by showing zeros.